### PR TITLE
Preventing making the document 'dirty' during initialization

### LINKED ISSUE #65: Save dialog prompt pops up even if nothing was changed
Open OLW, then close it immediately, without changing anything. You'll get the "unsaved changes" dialog. It shouldn't appear until the title or the content was changed.

### PR DESCRIPTION
**Pull request by hmemcpy**
This fixes #65.

Sometime during the completion callback for the document, **something** is mshtml is causing `IHtmlChangeSink.Notify` event to fire, causing setting the document to 'dirty', even if nothing really was changed. This PR fixes the problem by suppressing handling the Notify event during the completion callback. For more information, see the original issue #65.

### CODE REVIEW

File: src/managed/OpenLiveWriter.Interop.Mshtml/mshtml/IHTMLChangeLog.cs [hunk 1]
```diff
@@ -11,7 +11,7 @@ namespace mshtml
     public interface IHTMLChangeLog
     {
         [MethodImpl(MethodImplOptions.InternalCall, MethodCodeType = MethodCodeType.Runtime)]
-        void GetNextChange([In] ref byte pbBuffer, [In] int nBufferSize, out int pnRecordLength);
+        int GetNextChange([In] IntPtr pbBuffer, [In] int nBufferSize, out int pnRecordLength);
```
Comment (ScottIsAFool) [inline src/managed/OpenLiveWriter.Interop.Mshtml/mshtml/IHTMLChangeLog.cs]: Is this change required for this PR?

Comment (hmemcpy) [inline src/managed/OpenLiveWriter.Interop.Mshtml/mshtml/IHTMLChangeLog.cs]: @ScottIsAFool ah, this is a leftover from my hunt to diagnose the problem... It's not required for this PR, no, but it's a corrected signature. I can remove it.
